docs/topics/Threads.md

```diff
@@ -133,35 +136,57 @@ There are four new `GET` routes for enumerating threads in a specific channel:
 
 Webhooks can send messages to threads by using the `thread_id` query parameter. See the [execute webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook) docs for more details.
 
-## Additional context on the the THREAD_LIST_SYNC and THREAD_CREATE dispatches
+## Details about Thread Access and Syncing 
+
+While the syncing of threads is similar to channels, there are two important differences that are relevant for [Thread List Sync](#DOCS_TOPICS_GATEWAY_EVENTS/thread-list-sync) and [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) events:
+
+1. The [Gateway](#DOCS_TOPICS_GATEWAY) will only sync threads that the app has permission to view
+2. The Gateway will only sync threads once the app has "subscribed" to the guild. For context, in Discord's official clients, a subscription happens when the user visits a channel in the guild.
+
+These differences mean there is some unique behavior that is worth going into.
+
+### Thread Access
+
+#### Gaining Access to Private Threads
 
-While threads are mostly similar to channels in terms of structure and how they are synced, there are two important product requirements that lead to differences in how threads and channels are synced. This section helps explain the behavior behind the [Thread List Sync](#DOCS_TOPICS_GATEWAY_EVENTS/thread-list-sync) and [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) dispatches by going over those problems and how they are solved.
+When an app is added to a private thread, it likely doesn't have that thread in memory yet since it doesn't have permission to view it.
 
-The two product requirements are: The gateway will only sync threads to a client that the client has permission to view, and it will only sync those threads once the client has "subscribed" to the guild. For context, in Discord's official clients, a subscription happens when the user visits a channel in the guild.
+Private threads are only synced to you if you are a member or a moderator. Whenever a user is added to a private thread, the Gateway also sends a [Thread Create](#DOCS_TOPICS_GATEWAY_EVENTS/thread-create) event. This ensures the client always has a non-null value for that thread.
 
-As mentioned, these lead to a couple of edge cases that are worth going into:
+> info
+> The `THREAD_CREATE` event is also sent when the user is a moderator (and thus would already have the channel in memory).
+
+#### Gaining Access to Public Threads
+
+When a Discord client is added to a public thread but has not yet subscribed to threads, it might not have that public thread in memory yet. However, upon connecting, the Gateway will auto-subscribe apps (which are a type of client) to all thread events and active threads.
```

Comment:
I'm not sure if it was clear or not, but this section isn't really a "problem" for apps. It's more here just to explain why apps might experience some "unexpected" dispatches. Apps are already auto-subscribed to threads, so they don't have this problem that our clients do, but IIRC it wasn't trivial to determine that where I needed to in the backend, so everyone gets this extra dispatch.

So, this change is a bit weird because it leaves out all the explanation for why this happens. 

If you want to stick with this format though, maybe something more like:  "When a non-app is added to a public thread but has not yet "subscribed" to threads, it might not have that public thread in memory yet, which is a requirement for Discord's clients. To solve this, the Gateway will send a Thread Create event. Apps will receive this event as well, even though it is not necessary since the Gateway will auto-subscribe apps to all thread events and active threads."